src/main/java/com/blogify/blogapi/file/S3Service.java

```diff
@@ -32,22 +35,38 @@ public String uploadObjectToS3Bucket(String key, byte[] file) {
     }
   }
 
+  public String deleteS3Object(String key) {
+    DeleteObjectRequest deleteObjectsRequest = DeleteObjectRequest.builder().bucket(bucketName).key(key).build();
+    try {
+      s3Client.deleteObject(deleteObjectsRequest);
+      return key;
+    } catch (AwsServiceException | SdkClientException e) {
+      s3ErrorHandler(e);
+      return "error";
+    }
+  }
+
   public byte[] getObjectFromS3Bucket(String key) {
     GetObjectRequest objectRequest = GetObjectRequest.builder().bucket(bucketName).key(key).build();
     try {
       ResponseBytes<GetObjectResponse> objectBytes = s3Client.getObjectAsBytes(objectRequest);
-
       return objectBytes.asByteArray();
     } catch (AwsServiceException | SdkClientException e) {
-      int statusCode = 500;
-      if (e instanceof AwsServiceException awsException) {
-        statusCode = awsException.statusCode();
-      }
-      if (400 <= statusCode && statusCode < 500) {
-        throw new ApiException(ApiException.ExceptionType.CLIENT_EXCEPTION, e.getMessage());
-      } else {
-        throw new ApiException(ApiException.ExceptionType.SERVER_EXCEPTION, e.getMessage());
-      }
+      s3ErrorHandler(e);
+      return new byte[0];
+    }
+  }
+
+  private void s3ErrorHandler(Exception e){
```

Comment:
If you want to use this method, try `Thowable` as returned value, hence you can throw it